inst/webserver_setup/shinyproxy/application.yml

```diff
@@ -10,19 +10,24 @@ proxy:
   heartbeat-timeout: 6000 # if the server does not receive a heartbeat for heartbeat-timeout milliseconds, the relevant proxy will be released (60s as default)
   hide-navbar: true
   users:
-  - name: jack
-    password: password
-    groups: admins
+    - name: jack
+      password: password
+      groups: admins
   docker:
     internal-networking: true
   specs:
-  - id: deconvexplorer
-    display-name: DeconvExplorer
-    description: Shiny App for the omnideconv package
-    container-cmd: ["R", "-e", "shiny::runApp('/srv/shiny-server/', host = '0.0.0.0', port = 3838)"]
-    container-network: deconvexplorer_sp-net
-    container-image: deconv_explorer:latest
-    #container-cpu-limit: 2
+    - id: deconvexplorer
+      display-name: DeconvExplorer
+      description: Shiny App for the omnideconv package
+      container-cmd:
+        [
+          "R",
+          "-e",
+          "shiny::runApp('/srv/shiny-server/', host = '0.0.0.0', port = 3838)",
+        ]
+      container-network: webserversetup_sp-net-deconvexplorer
```

Comment:
Did you test the shinyproxy setup with the new network names? It is really sensitive to directory names and upper/lower case

Reply:
not really - I could not really have it set up in the short time. Could you give it a spin "as it is" from this branch?